packages/core/crates/core-crypto/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+pub mod checksum;
```

Comment:
Please move `checksum` related stuff to `Address`, no need to create an additional module for it imho.
Also please bump the version of the `core-crypto` crate.

Reply:
Will bump the version.